graphql/README.md

```diff
@@ -29,7 +29,17 @@ Entry point is `src/adapters/webserver/main.py`, which will start the applicatio
 - `graphql` is run via the `webserver` package.
 
 ## Deployment
-todo
+Compiled packages are stored at the [NASA Github packages page](https://github.com/orgs/nasa/packages/container/package/orca-graphql).
```

Comment:
Looks like it made me an admin when I uploaded the initial version. I've made it public... Does it work now?

Reply:
for naming convention, wouldn't we want to do something like `ghcr.io/nasa/cumulus-orca/graphql:version` so that items are under a common name for our stuff per https://nasa.github.io/cumulus-orca/docs/developer/research/research-pushing-docker-image

Reply:
Changed